modules/example/lua/example_lua_module.lua

```diff
@@ -0,0 +1,14 @@
+-----------------------------------
+-- Old setting from settings.lua: HOMEPOINT_HEAL
+-- Set if you want Home Points to heal you like in single-player Final Fantasy games.
+-----------------------------------
+require("modules/module_utils")
+require("scripts/globals/homepoint")
+-----------------------------------
+local m = Module:new("homepoint_heal")
+
+m:addOverride("xi.homepoint.onTrigger", function(player, csid, index)
+    super(player, csid, index)
```

Comment:
Just noticed this, but looks like this module doesn't apply any additional function (not sure if we want to have homepoint heal actually implemented)

Reply:
Good catch, I'm not sure what to leave in the Lua example, if anything